mobile_verifier/src/reward_shares.rs

```diff
@@ -84,14 +196,14 @@ impl OwnerShares {
             })
     }
 
-    pub fn into_radio_shares(
+    pub fn into_radio_shares<'a>(
         self,
-        epoch: &'_ Range<DateTime<Utc>>,
-    ) -> Result<impl Iterator<Item = proto::RadioRewardShare> + '_, ClockError> {
+        transfer_rewards: &'a TransferRewards,
+        epoch: &'a Range<DateTime<Utc>>,
+    ) -> Result<impl Iterator<Item = proto::RadioRewardShare> + 'a, ClockError> {
         let total_shares = self.total_shares();
-        let total_rewards = get_scheduled_tokens(epoch.start, epoch.end - epoch.start)?;
-        let rewards_per_share = (total_rewards / total_shares)
-            .round_dp_with_strategy(REWARDS_PER_SHARE_PREC, RoundingStrategy::ToPositiveInfinity);
+        let total_poc_rewards = transfer_rewards.remaining_poc_rewards;
```

Comment:
this doesn't seem right. shouldn't the total PoC rewards be 20% of that TOTAL_EMISSIONS_POOL number + any remainder from transfer rewards?